OrbitGl/TimeGraph.cpp

```diff
@@ -211,6 +211,31 @@ void TimeGraph::PanTime(int a_InitialX, int a_CurrentX, int a_Width,
   NeedsUpdate();
 }
 
+void TimeGraph::HorizontallyScrollIntoView(const TextBox* text_box) {
+  const Timer& timer = text_box->GetTimer();
+
+  if (IsVisible(timer)) {
```

Comment:
I feel this is a bit inconsistent. If a timer is not visible, this method will put the textbox, on the 1/3 line in the view, if it is visible it will stay wherever it is. Is there a reason you don't want to always move such that the textbox is on the 1/3 line? You'd probably want to rename the method if you change it.

Reply:
Let me explain it a little bit.
- Actually, we are moving horizontally as minimum as possible to see the text box completely. So, if the text box wasn't visible is always going to be in a side of the screen.
- On the other hand, if you press ctrl-f to find something in a google docs, it isn't going to move the screen if the word is visible (as we do), but if it isn't visible, the screen moves more than the minimum. It's nicer. In fact, it's a bit more complex how it moves, but the plan here if to make a general function which scrolls automatically to the left or the right depending on the text box position and also scrolling more than the minimum.

Reply:
Ah, I recall we discussed this. Sounds good to me. Would it make sense to move the logic for checking if the timer is visible up a bit? So that this function will always move a box to the same place in the view, no matter where it is, but the UI logic implements the behavior you suggest?